doc/rados/operations/erasure-code.rst

```diff
@@ -61,6 +61,31 @@ the following profile can be defined::
     $ rados --pool ecpool get NYAN -
     ABCDEFGHI
 
+Formula for calculating the raw size percentage overhead::
+
+    K/(K+M) * 100
+
+For example for profile (*K=3*) and (*M=2*) the percentage overhead is 60%.
+
+Formula for calculating the raw size overhead::
+
+    K/(K+M) * (total raw size)
+
+For example for profile (*K=3*) and (*M=2*) for total raw size 256T, the raw size overhead is 153.6T.
+
+Formula for calculating the plain used overhead::
```

Comment:
I think "plain used" is here to create a different term from the above. But I think you can get rid of "plain used" and just call this "overhead".